app.json

```diff
@@ -128,6 +128,10 @@
       "default": "MicroMasters",
       "description": "Application identifier in New Relic."
     },
+    "NODE_MODULES_CACHE": {
+      "description": "If false, disables the node_modules cache to fix yarn install",
+      "value": false
```

Comment:
`env` values always need to be strings: https://devcenter.heroku.com/articles/app-json-schema#env